experimental/devices/inky/inky.go

```diff
@@ -63,7 +106,16 @@ type Opts struct {
 	BorderColor Color
 }
 
-// New opens a handle to an Inky pHAT.
+const spiChunkSize = 4096
```

Comment:
I think this code was specifically designed to work around the default fairly small SPI driver buffer size on Raspbian.

It can now be discovered dynamically with MaxTxSize().
https://github.com/google/periph/blob/master/host/sysfs/spi.go#L141

It's useful to use this value because it uses the current value as exposed by the driver
https://github.com/google/periph/blob/master/host/sysfs/spi.go#L592

I see the documentation is currently lacking, it's only noted in the Conn comment
https://godoc.org/periph.io/x/periph/conn/spi#Conn

There's a formal interface https://godoc.org/periph.io/x/periph/conn#Limits but I think I'll put it in all busses for v4.